app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -362,8 +362,28 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
+      state_attributes = parse_container_state container.lastState
+      state_attributes.each { |key, val| h[key.to_s.prepend('last_').to_sym] = val } if state_attributes
+      h.merge! parse_container_state container.state
+    end
+
+    def parse_container_state(state_hash)
```

Comment:
Heh, if what I'm saying is at odds with Rubocop, forget it.  It's probably not worth battling the authorities over :)